pipeline/07-export.R

```diff
@@ -271,8 +271,40 @@ assessment_card <- dbGetQuery(
 #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 message("Preparing data for Desk Review export")
 
+# Get the number of apartments for class 211 and 212 properties. Since
+# char_apts is recorded on the card level, we need to aggregate them to
+# the PIN level
+num_apts_by_pin <- assessment_card %>%
+  filter(char_class == "211" | char_class == "212") %>%
+  select(meta_pin, char_apts) %>%
+  # Convert the long format for char_apts to a shorter one that's easier
+  # to scan in a spreadsheet
+  mutate(
+    char_apts = case_when(
+      char_apts == "None" | is.na(char_apts) ~ "Missing",
+      char_apts == "Two" ~ "2",
+      char_apts == "Three" ~ "3",
+      char_apts == "Four" ~ "4",
+      char_apts == "Five" ~ "5",
+      char_apts == "Six" ~ "6",
+      TRUE ~ "Missing"
+    )
+  ) %>%
+  # Adding the number of units might get confusing, since these classes are only
+  # supposed to have 2-6 units, so concatenate them as strings instead
+  summarize(char_apts = paste(char_apts, collapse = ", "), .by = "meta_pin")
+
+num_commercial_units_by_pin <- assessment_card %>%
+  filter(char_class == "212") %>%
+  select(meta_pin, char_ncu) %>%
+  summarize(char_ncu = paste(char_ncu, collapse = ", "), .by = "meta_pin")
```

Comment:
According to the assessment set I was working with, `char_ncu` is always present for 212s, so I didn't need to add the same handling for missing values that we use for `char_apts` above; however, this trend in the data might not always hold, so perhaps it would be worth adding some handling?

Reply:
I think it's fine for now. `char_ncu` is almost always populated (if only with `0`).